src/NuGet.Clients/NuGet.VisualStudio.Common/ProjectHelper.cs

```diff
@@ -26,19 +26,20 @@ public static async Task DoWorkInWriterLockAsync(Project project, IVsHierarchy h
                 var service = unconfiguredProject.ProjectService.Services.ProjectLockService;
                 if (service != null)
                 {
-                    using (ProjectWriteLockReleaser x = await service.WriteLockAsync())
-                    {
-                        await x.CheckoutAsync(unconfiguredProject.FullPath);
-                        ConfiguredProject configuredProject = await unconfiguredProject.GetSuggestedConfiguredProjectAsync();
-                        MsBuildProject buildProject = await x.GetProjectAsync(configuredProject);
-
-                        if (buildProject != null)
+                    await service.WriteLockAsync(
```

Comment:
Should I put all required source code changes in their own commit for easier review?